### PR TITLE
Recovery Services - added intros for heading sections

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/site-recovery.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-20/docs-ref-services/site-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180134494463-120%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/site-recovery.md [hunk 1]
```diff
@@ -21,15 +21,20 @@ Site Recovery allows you to automate replication of Azure VMs between regions, o
 
 Learn more about [Azure Site Recovery](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-overview).
 
-## Management Package
+## Management package
 
 ### Install npm module
+
+Install the Azure recovery servics npm module
```
Comment (clarkio) [inline docs-ref-services/site-recovery.md]: Typo in services. Also, according to the template it should be npm package: https://microsoft-my.sharepoint.com/personal/buhollan_microsoft_com/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bcd8cb267-e673-42b1-9d7b-0882e19caf27%7D

Comment (clarkio) [inline docs-ref-services/site-recovery.md]: Should we capitalize all product names? Should this be `Install the Azure Site Recovery npm package`?

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/site-recovery.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-20/docs-ref-services/site-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180355510176-120%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/site-recovery.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-20/docs-ref-services/site-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180359522650-120%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/site-recovery.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-20/docs-ref-services/site-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180412094578-120%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa @clarkio merged

**Pull request merged**